src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -132,6 +132,11 @@ class NoTemplatesServiceApi(object):
         and trailing
         whitespace.
 
+        Example:
+          >>> from google.example.v1.no_path_templates_pb2 import NoTemplatesServiceApi
```

Comment:
Ah, sorry, I see that you had already responded above that you'll look at this. Ignore that comment.